src/views/acquirer/data-required.njk

```diff
@@ -0,0 +1,35 @@
+{% extends "page.njk" %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form method="post">
+      {{ govukTextarea({
+            id: "data-required",
+            name: "data-required",
+            label: {
+                text: "What data from "  + " do you need?",
```

Comment:
I guess this is where you'd include the asset title?
Are you going to add that in a separate PR?
Might as well add it to this PR I guess, so we don't forgot to come back and update it?